framework/utils/InputData.py

```diff
@@ -584,9 +584,9 @@ def generateLatex(cls, recDepth=0):
         msg += '\n{sub}'.format(sub=sub.generateLatex(recDepth=recDepth+2))
       msg += '{i}\\end{{itemize}}\n'.format(i=doDent(recDepth, 1))
     # TODO is this a good idea? -> disables underscores in math mode :(
-    if recDepth == 0:
-      # assure underscore is escaped, but not doubly
-      msg = re.sub(r'(?<!\\)_', r'\_', msg)
+    #if recDepth == 0:
```

Comment:
I will remove the commented lines once Paul confirms this doesn't ruin something else

Reply:
wait, are we just removing the escaping of underscores entirely?

Reply:
Hi @PaulTalbot-INL , could you check it if these modifications work for you?

Reply:
We need these lines to build the manual in HERON correctly for sure, and we will need it for RAVEN if we ever want to build the whole manual dynamically in the future.

Reply:
@joshua-cogliati-inl, any thoughts?

Reply:
Is it possible that the complex multiline equations would be a better fit for the theory guide than the user manual?

Reply:
In constrained optimization, the fitness is defined in two cases, a function for individuals that do not violate constraints and another penalized function when individuals violate constraints. This is true for all the fitness functions implemented so far for constrained GA. Now in the user manual how will the user select between the fitness options? There has to be a minimal amount of information for the user to understand what should he/she provide.

Reply:
Yes, my thought is what in HERON needs these lines?  (Or at least provide an example of what will fail.) (As in I understand why name needs to be escaped, but I don't understand why the description can't do the escaping manually.)

Reply:
I went and built the manual with and without these changes; it definitely fails with the changes.

HERON dynamically reads options for entries (that go into the description of nodes) from the code base for the documentation, and this includes entries with underscores. These underscores have to be escaped to compile the tex.

Reply:
Hm, how does HERON dynamically read options? Can we escape the options at that point before it generates the the description?

Reply:
A quick count suggests about 86 breaking errors introduced by this change. I didn't look at all of them, but several are related to setting the acceptable types of ValuedParams that can be used to define the value of things like capacities, minimums, etc throughout the HERON input. It should be possible that this process could be modified to escape the underscores there. Then we can find any other breaks introduced.

I don't have anyone available that could do this right now, but some time in the coming months I'm sure it could be done.